_posts/2018-11-16-scalafix-scalameta.md

```diff
@@ -0,0 +1,136 @@
+---
+category: blog
+by: Ólafur Páll Geirsson, Eugene Burmako
+title: "Build code analyzers with Scalameta and Scalafix"
+---
+
+We are excited to announce the release of
+[Scalameta v4.0.0](https://github.com/scalameta/scalameta/releases/tag/v4.0.0)
+and
+[Scalafix v0.9.0](https://github.com/scalacenter/scalafix/releases/tag/v0.8.0-RC1)
```

Comment:
This points to v0.8.0-RC1

Reply:
Huh? I don't understand. (And, everyone will think it's a mistake, regardless of what the explanation is.)

Reply:
I copy pasted the v0.8.0-RC1 release notes to the v0.9.0 release to avoid the need to explain :)